components/widget/widget_table_cell.lua

```diff
@@ -0,0 +1,47 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Table/Cell
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Widget = require('Module:Infobox/Widget')
+
+local TableCell = Class.new(
+	Widget,
+	function(self, input)
+		self.content = input.content or {}
+		self.classes = input.classes or {}
+		self.css = input.css or {}
+	end
+)
+
+function TableCell:addContent(text)
+	table.insert(self.content, text)
+	return self
+end
+
+function TableCell:addClass(class)
+	table.insert(self.classes, class)
+	return self
+end
+
+function TableCell:make()
+	local cell = mw.html.create('div')
+	cell:css('border-left', '1px solid #bbb'):css('border-top', '1px solid #bbb'):css('background', 'inherit')
+	cell:css('padding', '5px'):css('line-height', '1.42857143')
```

Comment:
What is this line height value?

Reply:
I'm guessing it comes from
https://stackoverflow.com/questions/19982651/why-does-bootstrap-set-the-line-height-property-to-1-428571429

It's migrated from https://liquipedia.net/commons/index.php?title=MediaWiki:Common.css/DivTable.css&action=edit#mw-ce-l28

But it's also used in some other places
https://liquipedia.net/commons/index.php?search=insource%3A%221.42857143%22&title=Special%3ASearch&profile=advanced&fulltext=1&ns8=1

Though, after some additional testing. It should be removed.